how-to-guides/arbitrum-integration.md

```diff
@@ -47,7 +47,7 @@ Then the `Read` logic takes care of taking the deserialized Blob Pointer struct
 
 The following represents a non-exhaustive list of considerations when running a Batch Poster node for a chain with Celestia underneath:
 - You will need to use a consensus node RPC endpoint, you can
-[find a list of them for Mocha](../how-to-guides/mocha-testnet#community-rpc-endpoints)
+[find a list of them for Mocha](../how-to-guides/mocha-testnet.md#community-rpc-endpoints)
```

Comment:
Lychee and other link checkers usually choke on finding files w/o the extension, but np as vitepress can handle either. So would suggest a contributing/style guide mentions to use a particular format. I would suggest `[text](/base-working-dir/subdir/page.md#id)` for best chance at working longer term & compatibility

Reply:
do you want to add this to the README as a part of this PR?

Reply:
323a499116bb47cb951dafb88ae85eaffedf9e5e